src/data/finset/lattice.lean

```diff
@@ -1112,6 +1112,10 @@ lemma sup_eq_bUnion {α β} [decidable_eq β] (s : finset α) (t : α → finset
   s.sup t = s.bUnion t :=
```

Comment:
Ah, this `sup_eq_bUnion` lemma is about `finset.bUnion`: the one I was suggesting is about `set`s "bUnion" - so we should probably have both!

Reply:
Yes. And same for `sInter`, I presume?